internal/cli/atlas/privateendpoints/regionalmodes/descriptions.go

```diff
@@ -0,0 +1,22 @@
+// Copyright 2021 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package regionalmodes
+
+const (
+	regionalMode = "Manage regionalized private endpoint setting for one Atlas project."
```

Comment:
[nb] we usually say, "...your Atlas project..." rather than "one" but not blocking on this